src/modules/async.js

```diff
@@ -0,0 +1,8 @@
+import fetchData from './fetch.js';
+
+async function displayScore() {
```

Comment:
- [ ] Kindly, ensure the usage of arrow functions instead of the function keyword

Reply:
okay, thanks